test/api/OperatorApi/orderStatus.test.ts

```diff
@@ -0,0 +1,208 @@
+    })
+    test('Filled, surplus', () => {
+      const order: RawOrder = { ...BASE_ORDER, kind: 'buy', buyAmount: '1000', executedBuyAmount: '1001' }
+
+      expect(getOrderStatus(order)).toEqual('filled')
+    })
+  })
+  describe('Sell order', () => {
+    test('Not filled', () => {
+      const order: RawOrder = { ...BASE_ORDER, kind: 'sell', sellAmount: '10000', executedSellAmount: '9998' }
+
+      expect(getOrderStatus(order)).not.toEqual('filled')
+    })
+    test('Filled, within epsilon', () => {
+      const order: RawOrder = { ...BASE_ORDER, kind: 'sell', sellAmount: '10000', executedSellAmount: '9999' }
+
+      expect(getOrderStatus(order)).toEqual('filled')
+    })
+    test('Filled, exact amount', () => {
+      const order: RawOrder = { ...BASE_ORDER, kind: 'sell', sellAmount: '100', executedSellAmount: '100' }
+
+      expect(getOrderStatus(order)).toEqual('filled')
+    })
+    test('Filled, surplus', () => {
+      const order: RawOrder = { ...BASE_ORDER, kind: 'sell', sellAmount: '1000', executedSellAmount: '1001' }
+
+      expect(getOrderStatus(order)).toEqual('filled')
+    })
+  })
+})
+
+describe('Partially filled status', () => {
+  describe('Buy order', () => {
+    test('Not partially filled, fully filled', () => {
+      const order: RawOrder = { ...BASE_ORDER, kind: 'buy', buyAmount: '10000', executedBuyAmount: '10000' }
+
+      expect(getOrderStatus(order)).not.toEqual('partially filled')
+    })
+    test('Not partially filled, no fills', () => {
+      const order: RawOrder = { ...BASE_ORDER, kind: 'buy', buyAmount: '10000', executedBuyAmount: '0' }
+
+      expect(getOrderStatus(order)).not.toEqual('partially filled')
+    })
+    test('Partially filled', () => {
+      const order: RawOrder = { ...BASE_ORDER, kind: 'buy', buyAmount: '10000', executedBuyAmount: '11' }
+
+      expect(getOrderStatus(order)).toEqual('partially filled')
+    })
+  })
+  describe('Sell order', () => {
+    test('Not partially filled, fully filled', () => {
+      const order: RawOrder = { ...BASE_ORDER, kind: 'sell', sellAmount: '10000', executedSellAmount: '10000' }
+
+      expect(getOrderStatus(order)).not.toEqual('partially filled')
+    })
+    test('Not partially filled, no fills', () => {
+      const order: RawOrder = { ...BASE_ORDER, kind: 'sell', sellAmount: '10000', executedSellAmount: '0' }
+
+      expect(getOrderStatus(order)).not.toEqual('partially filled')
+    })
+    test('Partially filled', () => {
+      const order: RawOrder = { ...BASE_ORDER, kind: 'sell', sellAmount: '10000', executedSellAmount: '11' }
+
+      expect(getOrderStatus(order)).toEqual('partially filled')
+    })
+  })
+})
+
+describe('Expired status', () => {
+  beforeEach(mockTimes)
+
+  describe('Buy order', () => {
+    test('Not expired, open', () => {
+      const order: RawOrder = {
+        ...BASE_ORDER,
+        kind: 'buy',
+        buyAmount: '10000',
+        executedBuyAmount: '0',
+        validTo: Math.floor(Date.now() / 1000),
```

Comment:
maybe you can use a aux function `_getPastDate()` and `_getFutureDate()`? just to read the case easier?.